src/com/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -57,7 +57,9 @@
             [com.puppetlabs.puppetdb.config :as conf]
             [com.puppetlabs.puppetdb.utils :as utils]
             [puppetlabs.kitchensink.core :as kitchensink]
+            [robert.hooke :as rh]
             [puppetlabs.trapperkeeper.core :refer [defservice main]]
+            [puppetlabs.trapperkeeper.services :refer [service-id]]
```

Comment:
Oh funny, so you didn't add any new references to service-id other than here, but the code before worked. We must be getting it from the anaphora in  the service macro.